index.js

```diff
@@ -1,5 +1,57 @@
-$(document).ready(function () {
-  $(".dropdown-btn").click(function () {
-    $(".dropdown-content").toggle("close");
-  });
+const MobileNav = document.querySelector('.mobile-nav');
+const NavbarContent = document.querySelector('.navbar-content');
+const MobileNavImg = document.querySelector('.mobile-nav-img');
+
+const IconClose = document.querySelector('.icon-close');
+
+const NavbarOptionsPortfolio = document.querySelector(
+  '.navbar-options-portfolio',
+);
+const NavbarOptionsAbout = document.querySelector('.navbar-options-about');
+const NavbarOptionsContact = document.querySelector('.navbar-options-contact');
+
+const HeadlineWrapper = document.querySelector('.headline');
+const logo = document.querySelector('.logo');
+// const header = document.querySelector("header");
+
+function blurbackground() {
```

Comment:
- [ ] Kindly remove this commented-out code in order to make your codebase cleaner. 👍